src/assisted_installer_controller/assisted_installer_controller_test.go

```diff
@@ -187,11 +189,9 @@ var _ = Describe("installer HostRoleMaster role", func() {
 	}
 
 	setControllerWaitForOLMOperators := func(clusterID string) {
-		WaitTimeout = 100 * time.Millisecond
-
 		setClusterAsFinalizing()
-		uploadIngressCert(clusterID)
```

Comment:
why to move it?

Reply:
since the console availability now happens before uploading the ingress certificate.